nixos/modules/services/web-apps/grav.nix

```diff
@@ -111,6 +111,7 @@ in
               exif
               filter
               gd
+              intl
```

Comment:
The `intl` extension is enabled by default, shouldn't you use the `enabled` parameters and append the extensions you need?

Reply:
e.g., something like: `enabled ++ [ apcu xml yaml ]` ?

Reply:
Awesome, that works just fine. I've updated the PR to do this now. Thanks for the help!